lib/util.js

```diff
@@ -98,17 +98,17 @@ function getDownloadSpec() {
   var platform = getTargetPlatform()
   var arch = getTargetArch()
   if (platform === 'linux' && arch === 'x64') {
-    downloadUrl += 'linux-x86_64.tar.bz2'
-    checksum = '86dd9a4bf4aee45f1a84c9f61cf1947c1d6dce9b9e8d2a907105da7852460d2f'
+    downloadUrl += 'linux-x86_64.tar.gz'
```

Comment:
where does this come from? AFAICT they're still struggling to come up with static linux binaries, and had to do per-distro binaries, cf.
https://bitbucket.org/ariya/phantomjs/downloads